src/EnvAccessor.php

```diff
@@ -31,9 +31,11 @@ public function get($key, $default = null)
             if ($this->localOnly) {
                 trigger_error('localOnly option is only available in PHP 7');
             }
-            return getenv($key) ?: $default;
+            $value = getenv($key);
+            return $value === false ? $default : $value;
         }
-        return getenv($key, $this->localOnly) ?: $default;
+        $value = getenv($key, $this->localOnly);
+        return $value === false ? $default : $value;
```

Comment:
Problem with this is that explicitly specified false in environment variable will still return `$default`.
That will be bad for cases like `@env::get('ALLOW_SOMETHING', true)` - specifying `false` in environment will result in `true`.